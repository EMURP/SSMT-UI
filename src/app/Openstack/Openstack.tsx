import * as React from 'react';
import { PageSection, Title }  from '@patternfly/react-core';
// import { ChartPie, ChartThemeColor } from '@patternfly/react-charts';
import { Fetchdata } from '@app/fetchdata/fetchdata';
import { FilterData } from '@app/fetchdata/filterData';
import { SearchData } from '@app/fetchdata/searchData';
import { ExpandableDiv } from '@app/Expandable/ExpandableDiv';


const Openstack: React.FunctionComponent<{}> = () => (


  <PageSection>


    <Title headingLevel="h1" size="lg">Openstack </Title>
    <ExpandableDiv/>
  {/* <PaginationTop/> */}
    
  </PageSection>
)

export { Openstack };