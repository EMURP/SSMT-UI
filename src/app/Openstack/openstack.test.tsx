import React from 'react';
import { shallow } from 'enzyme';
import { Openstack } from './Openstack';

describe('Openstack tests', () => {

  test('should render default Openstack component', () => {
    const view = shallow(<Openstack />);
    expect(view).toMatchSnapshot();
  })

})