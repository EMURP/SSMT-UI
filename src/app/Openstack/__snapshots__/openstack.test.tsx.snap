// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Openstack tests should render default Openstack component 1`] = `
<Component>
  <Component
    headingLevel="h1"
    size="lg"
  >
    Openshift Cluster Data
  </Component>
</Component>
`;
